materials/functions-R.md

```diff
@@ -158,4 +158,4 @@ est_shrub_mass_dim <- function(length, width, height){
 est_shrub_mass_dim(0.8, 1.6, 2.0)
 ```
 
-> Do [Nested Functions]({{ site.baseurl }}/exercises/Functions-nested-functions-R).
+> Do [Combined Functions]({{ site.baseurl }}/exercises/Functions-combined-functions-R).
```

Comment:
This link is still broken, it's supposed to be Functions-combining-functions-R, I think?

Reply:
Argh. Sorry. I've been having some git issues and apparently I didn't refix that in the mess. I'm going to go ahead and merge and then update shortly (for reasons related to the ongoing git issues).